spec/enumerable_spec.rb

```diff
@@ -1,11 +1,15 @@
-require '../enumarable.rb'
+# rubocop:disable Style/EvenOdd
+# rubocop:disable Layout/LineLength
+
+require './enumarable.rb'
 
 describe 'Enumerable' do
-  describe '#my_each' do
-    arr = [1, 2, 3, 4]
-    hash = { 'a' => 1, 'b' => 2, 'c' => 3 }
-    range = (1..4)
+  arr = [1, 2, 3, 4]
+  str = %w[ant bear cat]
+  hash = { 'a' => 1, 'b' => 2, 'c' => 3 }
+  range = (1..4)
```

Comment:
- [ ] Rspec has a specific way of declaring variables please review this [LINK](https://github.com/rubocop-hq/rspec-style-guide#let-blocks)